proto-lens-protoc/app/Data/ProtoLens/Compiler/Editions/Defaults.hs

```diff
@@ -0,0 +1,45 @@
+-- Copyright 2024 Google LLC. All Rights Reserved.
+--
+-- Use of this source code is governed by a BSD-style
+-- license that can be found in the LICENSE file or at
+-- https://developers.google.com/open-source/licenses/bsd
+
+{-# LANGUAGE OverloadedLabels #-}
+
+{-|
+Module: Data.ProtoLens.Compiler.Editions.Defaults
+Description: Exports defaults for native features in Protobuf Editions.
+Copyright: Copyright (c) 2024 Google LLC
+License: BSD3
+-}
+module Data.ProtoLens.Compiler.Editions.Defaults (nativeDefaults) where
+
+import Data.ByteString (ByteString)
+import Data.ProtoLens (decodeMessage)
+import Proto.Google.Protobuf.Descriptor (FeatureSetDefaults)
+
+{-| 'FeatureSetDefaults' message containing feature defaults.
+
+This contains the defaults from editions @LEGACY@ to @EDITION_2023@
+for the native features defined by @google.protobuf.FeatureSet@.
+-}
+nativeDefaults :: FeatureSetDefaults
+nativeDefaults
+  | Right m <- msg = m
+  | Left e <- msg = error $ "unable to decode built-in defaults: " ++ e
+  where msg = decodeMessage serializedNativeDefaults
```

Comment:
Why not just build the proto message with regular Haskell code rather than decode a binary string embedded in a source file...?

Reply:
I see, you addressed this in the PR description:

> it does not seem sustainable to manually translate the protobuf message to Haskell code every time there is a new edition

Hm, really? The proto is pretty small, at least right now. My concern is that it makes really hard to tell what the values actually are (e.g. for debugging). I think it's better to have the proto generated by Haskell code, but it's up to you.

Reply:
"Not sustainable" was really just another way of saying "an annoying amount of manual work".  I would rather trust the output of `protoc` instead of including complicated instructions to generate the equivalent Haskell code.

(Truth be told, I wasn't actually able to get the correct incantations for either `ghci` or `stack ghci` to even attempt a `show` for the `FeatureSetDefaults` message, assuming that `show` for protobuf messages even outputs valid Haskell code.)